src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -13,37 +13,22 @@
 
 public class AutoplayVuap {
 
-  private static final String SLOT_SELECTOR_PREFIX = "#%s .";
-  private static final String PAUSE_CLASS_NAME = "pause-overlay";
-  private static final String REPLAY_CLASS_NAME = "replay-overlay";
-  private static final String CURRENT_TIME_CLASS_NAME = "current-time";
-  private static final String SPEAKER_CLASS_NAME = "speaker";
-  private static final String CLOSE_BUTTON_CLASS_NAME = "close-ad";
+  private static final String PAUSE_SELECTOR = ".pause-overlay";
+  private static final String REPLAY_SELECTOR = ".replay-overlay";
+  private static final String CURRENT_TIME_SELECTOR = ".current-time";
+  private static final String SPEAKER_SELECTOR = ".speaker";
+  private static final String CLOSE_BUTTON_SELECTOR = ".close-ad";
+  private static final String IMA_BRIGDE_SELECTOR = ".video-player iframe[src*='imasdk']";
+  private static final String VIDEO_SELECTOR = "video";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "video" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)